lib/active_model/serializer.rb

```diff
@@ -54,10 +57,11 @@ def self.type(type)
     end
 
     def self.attributes(*attrs)
+      options = attrs.last.class == Hash ? attrs.pop : {}
       attrs = attrs.first if attrs.first.class == Array
 
       attrs.each do |attr|
-        attribute(attr)
+        attribute(attr, options)
```

Comment:
I don't think we should support any options for `attributes`. That's always just a list.  We use `attribute` for an attribute with options